ruff.toml

```diff
@@ -5,12 +5,16 @@ extend-select = [
 	"W",
 
 	# local
+	"B",
+	"I",
 	"ISC",
 	"RUF010",
 	"RUF100",
 	"UP",
 ]
 ignore = [
+	"B028",
+	"B904",
```

Comment:
I think these should also be in a `# local` section below these upstream boilerplate settings.